main/ertp/api/issuer.md

```diff
@@ -1,214 +1,263 @@
 # Issuer
 
-The issuer cannot mint new amounts, but it can create empty purses and payments. The issuer can also transform payments (splitting payments, combining
-payments, burning payments, and claiming payments exclusively). The issuer should be obtained from a trusted source and then relied upon as the
-authority as to whether an untrusted payment is valid.
+An `issuer` represents the identity of a specific kind of digital asset. 
```

Comment:
I would say the brand is actually the identity of the kind of asset. The Issuer is the authority on what holds digital assets of this kind.